docs/fitting.rst

```diff
@@ -666,6 +666,10 @@ convenience functions to perform exactly this task.  An example is shown below.
     plt.title('Continuum Fitting')
     plt.grid('on')
 
+    # construct normalized spectrum by dividing the old spectrum
```

Comment:
This is a little buried with it being a comment in a code block. I think you should bring this out of the code block and demonstrate the results of the normalization with its own code block and plot.

Reply:
The problem is that you can't continue the code from one plot directive in another plot directive. If I regenerate a random spectrum a second time, it'll look different and we don't want the normalized spectrum to look different from the original one. Could set seed for random, but that's kind of not what we want. So, instead, I made it a second plot. the code is still in the same code block, but the fact that there is a second plot makes is a little more obvious to look there.

Reply:
Thank you for your advice. I thought I had a reasonable understanding of the plot directive given that I actually contributed a PR to matplotlib for one of the features, but apparently I was wrong. I will revise my doc change yet again.

Reply:
How about this version?
(Let me also note that there was nothing like this in the docs before. Even if my addition does not highlight it perfectly, I think it still makes the docs better.)